asv_bench/benchmarks/benchmarks.py

```diff
@@ -557,3 +557,8 @@ def time_columns(self, shape):
 
     def time_index(self, shape):
         return self.df.index
+
+    def time_dtypes(self, shape):
+        # trigger _compute_dtypes func
+        self.df._query_compiler._modin_frame._dtypes = None
+        return self.df.dtypes
```

Comment:
This makes bench strictly engine specific, can we somehow allow it to execute only for pandas backend?

Reply:
I think now it was enough to fix pandas case

Reply:
@devin-petersohn do you see a more general way to time this?

Reply:
This is useful to benchmark, maybe we should have a utility that wipes metadata caches?

Reply:
Create separate tracker for this https://github.com/modin-project/modin/issues/2780

There is no blocker for dtypes benchmark I suppose.